app.css

```diff
@@ -6,7 +6,299 @@ box-sizing: border-box;
 box-sizing: border-box;
 }
 
-
 body {
 margin: 0;
 }
+
+.header_container {
+background-image: url(images/Background-waves.jpg);
+background-size: 100% 100%;
+overflow: auto;
+position: relative;
+}
+
+.main__heading {
+  padding: 12rem;
+  text-align: center;
+  display: flex;
+}
+
+.nav-bar__container {
+  margin: auto;
+  max-width: 85%;
```

Comment:
Don't use percentages for `max-width`. Instead set a `width` in terms of percent and then `max-width` using REMs.

Also make sure you use the same method to double container the navbar, footer nav, and main content.